x11-libs/xcb-util/xcb-util-0.4.0-r2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+XORG_MULTILIB=yes
+inherit xorg-3
+
+DESCRIPTION="X C-language Bindings sample implementations"
+HOMEPAGE="https://xcb.freedesktop.org/ https://gitlab.freedesktop.org/xorg/lib/libxcb-util"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+
+RDEPEND=">=x11-libs/libxcb-1.9.1:=[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}"
```

Comment:
I'd suggest adding `x11-base/xorg-proto` to `DEPEND`. The reasoning is that

1) it's super cheap to install
2) it's in all likelihood already installed
3) there's a small possibility that somehow it's not installed and some user will figure out how to find that crack and fall into it